### DISCUSSION TITLE
Welcome to ecosystem-contributions Discussions!

### DISCUSSION THREAD

**Discussion by Nicca42** [ethereum-optimism/ecosystem-contributions#162] [Announcements]
## 👋 Hello!

Open a Discussion for each idea you want to build. The Discussion should be used to:
* Coordinate with other developers who want to help
* Post updates about your build
* Ask for help from fellow builders 
* Brag about any grants or Retro Funding you get! 

We recommend you build it open-sourced and under MIT licence. This makes it easier for other developers to step in and help! 

Each discussion MUST be related to an idea. 

## Reminders & Rules

**A reminder** that Builder Ideas need to have guaranteed funding.

Discussions opened to add ideas to the list will be closed. Multiple or spam discussions will also be closed. All interactions here must comply with the [Optimism Rules of Engagement](https://gov.optimism.io/t/rules-of-engagement-2-0/5728). Violations may result in you being removed from Optimism platforms. 

As always, stay optimistic, and we are excited to see what you build!

**Comment by wslyvh** [comment]
Hi @Nicca42 - love the new builder idea portal. Awesome work!

But I'm not sure how to propose or start any discussion for specific ideas? 
The link to "Join discussion" on the portal seem to linked to the pages itself. 
E.g. https://contribute.optimism.io/issue/event-ticket-app 

But would love to start, or join a discussion on the Event ticket app, so started one here. Hope that's the right way?
https://github.com/ethereum-optimism/ecosystem-contributions/discussions/168
